pkg/config/wikidata.go

```diff
@@ -74,23 +83,41 @@ var wikidata = struct {
 	// wikidatahome describes the name of the wikidata directory withing
 	// $SFHOME.
 	wikidatahome string
+	// wikibaseSparql is a placeholder for custom queries to be provided
+	// to a custom Wikibase instance.
+	wikibaseSparql string
+	// wikibaseSparql points to the wikibase.sparql file required to
+	// query a custom Wikibase endpoint.
+	wikibaseSparqlFile string
+	// wikibasePropsFile is a JSON file that stores all the properties
+	// needed for Roy to interpret a custom Wikibase SPARQL result.
+	wikibasePropsFile string
 	// wikibaseURL is the base URL needed by Wikibase for permalinks to
-	// resolve.
+	// resolve and for revision history to be retrieved. The URL will be
+	// augmented with /w/index.php or /w/api.php via Wikiprov (so do not
+	// add this!). Wikiprov is the package used to make revision history
+	// and permalinks happen.
 	wikibaseURL string
 }{
 	arc:                    "Q7978505",
 	arc1_1:                 "Q27824065",
 	gzip:                   "Q27824060",
 	tar:                    "Q283579",
 	warc:                   "Q10287816",
-	definitions:            "wikidata-definitions-2.0.0",
+	definitions:            "wikidata-definitions-3.0.0-wikibase",
```

Comment:
Updated - users will need to harvest again for this release. The API hasn't changed...